test/components/submission/list.spec.js

```diff
@@ -386,10 +353,7 @@ describe('SubmissionList', () => {
             props: { top: () => 2 }
           })
             .complete()
-            .request(component => {
-              sinon.replace(component.vm, 'scrolledToBottom', () => true);
-              document.dispatchEvent(new Event('scroll'));
-            });
+            .testNoRequest(scroll);
```

Comment:
Replacing `.request()` with `.testNoRequest()`. The two are basically aliases, and I think that makes the intent of the test clearer.